publish/creating-a-stack-hub/creating-a-stack-hub.md

```diff
@@ -132,15 +132,15 @@ In this guide, you will change the example configuration file and use it build a
               - java-microprofile
               - java-openliberty
               - nodejs
-          - url: https://github.com/kabanero-io/collections/releases/download/0.6.0/kabanero-index.yaml
+          - url: https://github.com/kabanero-io/collections/releases/download/0.9.0/kabanero-index.yaml
             include:
               - java-openliberty
     image-org:
     image-registry:
     nginx-image-name:
     ```
 
-    This example configuration file builds a single stack hub index file for your `default` hub. The `default` hub points to two index files, `https://github.com/appsody/stacks/releases/latest/download/incubator-index.yaml` and `https://github.com/kabanero-io/collections/releases/download/0.6.0/kabanero-index.yaml`. The `include:` and `exclude:` options are used
+    This example configuration file builds a single stack hub index file for your `default` hub. The `default` hub points to two index files, `https://github.com/appsody/stacks/releases/latest/download/incubator-index.yaml` and `https://github.com/kabanero-io/collections/releases/download/0.9.0/kabanero-index.yaml`. The `include:` and `exclude:` options are used
```

Comment:
the /collections URL isn't used anymore, this is the new format:
https://github.com/kabanero-io/kabanero-stack-hub/releases/download/0.9.0/kabanero-stack-hub-index.yaml
might need to do a global search in guides for this URL

Reply:
It is confusing, but if you look at the `kabanero-stack-hub-index.yaml` file, it still points to `https://github.com/kabanero-io/collections/releases/download/0.9.0/kabanero-index.yaml`.

The `kabanero-stack-hub-index.yaml` file is effectively what Steve and Matt generate with the process that this guide covers, so my thinking was that it should remain with the older URL for now ... that URL looks like it is still being generated for the 0.9.0 release. See https://github.com/kabanero-io/collections/releases/tag/0.9.0-rc.1.

@groeges or @kilnerm - are the updates I made for this guide ok this time around? i.e. guide combines appsody `latest` stacks with kabanero `0.9.0` stacks to generate a stack-hub yaml?

Reply:
Thanks for the clarification @SueChaplain . I'll approve as is.

Reply:
@jantley-ibm This config is what is used to generate the kabanero-stack-hub-index.yaml, so the URL cannot point to itself. The stacks are still coming from the Kabanero Collections repo as well as the Appsody repo to generate the consolidated index. We will need to continue using the collections repo until the certified stacks repo is available (now delayed to 3Q).